oreClient/src/main/resources/assets/Home.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+    <div class="row">
+        <div class="col-md-9">
+            <div class="project-search">
+                <input type="text" class="form-control" v-model="q" :placeholder="'Search in ' + projectCountPlaceholder + ' projects, proudly made by the community...'" />
```

Comment:
Do we want to update the project count when a filter is applied too?

Reply:
Isn't this updating? It's bound to the pagination.count of the last request. So if a filter is applied pagination.count should have a different value or not?

Reply:
What I more meant is if this is the behavior we want, or if we want the count to  only change when no filters are used.

Reply:
I'd say we should update it. Will push a commit later I've locally already which changes the message if filters are active but still updates the count.

Reply:
Can you recheck if this fits now?